package.json

```diff
@@ -22,20 +22,19 @@
     "decompress",
     "decompressplugin",
     "extract",
-    "gulpplugin",
     "tar"
   ],
   "dependencies": {
     "is-tar": "^1.0.0",
```

Comment:
Maybe we should just use the `file-type` module on all these plugins? That way it dedupes to one module for all plugins instead of many smaller ones.